Makefile

```diff
@@ -50,5 +56,9 @@ check: check-code stable-output
 install:
 	cargo install --locked --path sim-cli
 
+fix-all:
```

Comment:
`format` is probably a better name for this. Likewise would like a `CLIPPY_CMD` to de-duplicate.